dali/pipeline/executor/workspace_policy.h

```diff
@@ -152,12 +164,12 @@ inline void SetupStreamsAndEvents<OpType::GPU>(
 template <OpType op_type>
 op_type_to_workspace_t<op_type> CreateWorkspace(
     const OpGraph &graph, const OpNode &node,
-    const std::vector<tensor_data_store_queue_t> &tensor_to_store_queue,
+    const std::vector<tensor_data_store_queue_t> &tensor_to_store_queue, ThreadPool *thread_pool,
```

Comment:
weird line length

Reply:
That's how clang-format did it. The template parameters are looong, and it's hard to fit them. I can split it to separate lines, but that's not better by much.